### ISSUE TITLE
registration form validation

### ISSUE THREAD

**Issue by cpritcha** [virtualcommons/port-of-mars#446]
- [ ] check that both entered emails are identical
- [ ] check that emails are valid emails

**Comment by alee** [comment]
looks like there might be a server side issue:
```
server_1  | (node:40) UnhandledPromiseRejectionWarning: Error: 'to' parameter is not a valid address. please check documentation
server_1  |     at IncomingMessage.<anonymous> (/code/server/node_modules/mailgun-js/lib/request.js:327:17)
server_1  |     at IncomingMessage.emit (events.js:327:22)
server_1  |     at IncomingMessage.EventEmitter.emit (domain.js:547:15)
server_1  |     at endReadableNT (_stream_readable.js:1201:12)
server_1  |     at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

**Comment by alee** [comment]
fixed in 2d3c84db24efceb07c8cc435581fdc635ddb7120

it's hard to validate emails without just sending them, using HTML5 email input type for basic validation
